extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -0,0 +1,280 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "Python 3"
+        },
+        "language_info": {
+            "name": "python",
+            "version": "3.7.2",
+            "mimetype": "text/x-python",
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "pygments_lexer": "ipython3",
+            "nbconvert_exporter": "python",
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Jupyter Books\n",
+                "\n",
+                "## 1. Installation\n",
+                "\n",
+                "To install the Jupyter Book command-line interface (CLI), use `pip`!"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "97541c75-b1c9-4e4c-9f0a-f93df4a550ef"
+            }
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "import sys\r\n",
+                "\r\n",
+                "#install jupyter-book\r\n",
+                "cmd = f'{sys.executable} -m pip show jupyter-book'\r\n",
+                "cmdOutput = !{cmd}\r\n",
+                "if len(cmdOutput) > 0 and '0.6.4' in cmdOutput[1]:\r\n",
+                "    print('Jupyter-book required version is already installed!')\r\n",
+                "else:\r\n",
+                "    !pip install jupyter-book"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "8bd77173-2f63-4bf8-95e8-af2a654fc91e",
+                "tags": []
+            },
+            "outputs": [],
+            "execution_count": 1
```

Comment:
Clear results

Reply:
I cleared the results, what's expected here?